IFC4x3/Properties/i/IncomingStation_2ScYZDmQ9C9vR878Q0zliC/Documentation.md

```diff
@@ -0,0 +1 @@
+The station value for the incoming segment that ends at this referent location.
```

Comment:
Could probably delete 'referent' here as it is an abstract concept.

Reply:
Fixing, and also changing replacing "for" with "of" as with OutgoingStation